scripts/webpack/webpack.prod.js

```diff
@@ -117,6 +118,7 @@ module.exports = merge(common, {
       excludeChunks: ['manifest', 'dark', 'light'],
       chunksSortMode: 'none',
     }),
+    new BundleAnalyzerPlugin(),
```

Comment:
Not sure how much time this adds to the build time, but maybe we should have control over this i.e. via env variable? Bundle analyzer data will be useful for us especially to detect regression between releases. Other than that the analysis can be performed on demand. WDYT?

Reply:
If this produces something we can easily track using:
https://github.com/grafana/grafana/blob/master/scripts/ci-frontend-metrics.sh#L14

then I think it is OK to add by default.  If it is just the visualization, then we should add a flag for it and maybe an entry in the `package.json` scripts